Tests/test_file_ppm.py

```diff
@@ -49,6 +49,38 @@ def test_pnm(tmp_path):
         with Image.open(f) as reloaded:
             assert_image_equal(im, reloaded)
 
+def test_not_ppm(tmp_path):
+    path = str(tmp_path / "temp.djvurle")
+    with open(path, "wb") as f:
+        f.write(b"PyXX")
```

Comment:
If you change `PyXX` to something like `PyInvalid` you'll also cover the 6-byte length check.

Reply:
This was done in https://github.com/python-pillow/Pillow/pull/5121/commits/bc5ecfb79c3dca0553e2b863ba7d306f2e0fd434